examples/tic-tac-toe/server/server.bal

```diff
@@ -19,19 +19,38 @@ import ballerina/websocket;
 
 const string USER_SIGN = "userSign";
 const string SIGN_X = "X";
-const string SIGN_0 = "0";
+const string SIGN_O = "O";
```

Comment:
Just to confirm. It is "O" not "o" right.

Reply:
It is the letter capital O.